mxcube3/ui/actions/login.js

```diff
@@ -30,6 +30,19 @@ export function selectProposal(prop) {
   };
 }
 
+export function sendMail(sender, content) {
```

Comment:
`sender` is not used later in the server

Reply:
I'm now attaching the sender information the user provides to the subject of the mail